cmd/lakectl/cmd/local.go

```diff
@@ -71,6 +71,8 @@ func localDiff(ctx context.Context, client apigen.ClientWithResponsesInterface,
 	changes, err := local.DiffLocalWithHead(currentRemoteState, path, local.Config{
 		SkipNonRegularFiles: cfg.Local.SkipNonRegularFiles,
 		IncludePerm:         cfg.Experimental.Local.POSIXPerm.Enabled,
+		IncludeUID:          cfg.Experimental.Local.POSIXPerm.IncludeUID,
+		IncludeGID:          cfg.Experimental.Local.POSIXPerm.IncludeUID,
```

Comment:
Is this `IncludeUID` and not `IncludeGID` on purpose?

9Same q for the other files of course.)

Reply:
Thanks - good catch!